test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -1,13 +1,12 @@
 package seedu.addressbook.common;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-
 import java.util.Arrays;
 import java.util.List;
 
 import org.junit.Test;
 
+import static org.junit.Assert.*;
```

Comment:
Not permitted by the coding standard. Imported classes should always be listed explicitly.